dataModel.OSLO.PassengerTransportHubs/context.jsonld

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I need t create something for this especifically

Reply:
I have created a customization method for contexts. See the new version of [contribution manual](https://bit.ly/contribution_manual)

Reply:
Already implemented. please see the post on how to do it. create the notes_context.jsonld file [https://smartdatamodels.org/index.php/new-customization-option-for-context/](https://smartdatamodels.org/index.php/new-customization-option-for-context/)